src/routes/console/project-[project]/overview/platforms/wizard/flutter/step1.svelte

```diff
@@ -23,7 +23,13 @@
         Web = 'flutter-web'
     }
 
-    let platform: Platform = Platform.Android;
+    function isPlatform(value: string): value is Platform {
+        return Object.values(Platform).includes(value as Platform);
+    }
+
+    let platform: Platform = isPlatform($createPlatform.type)
+        ? $createPlatform.type
+        : Platform.Android;
```

Comment:
That does only seem to persist once 👍🏻 

https://user-images.githubusercontent.com/1759475/218757230-9e8b2116-caed-4b98-811f-5583e9ed0c9e.mov